src/aiida_quantumespresso/workflows/protocols/core_hole_treatments.yaml

```diff
@@ -0,0 +1,27 @@
+default_inputs:
```

Comment:
I think the plan here is to provide this for other classes/functions to access, notably the in-progress WorkChain for XPS (#879), though admittedly this is only meaningful for core-level spectroscopy calculations so it would make sense for it to go under `xspectra`

@superstar54, do you agree with all that?

Reply:
Since it will also be accessed by XPS (maybe other core-level spectroscopy methods in the future), I think we can keep it here instead of moving to xspectra.

Reply:
Ah, perhaps there is a slight misunderstanding. The XPS workflow itself actually just uses the PwBaseWorkChain and does not use `xspectra.x` to compute XPS - essentially all that happens there is we calculate the difference in total energy between `pw.x` calculations where the localised core-hole is on different symmetrically non-equivalent atoms of the same species.

The future plan involves expanding the theory further to calculate absolute values for the XPS binding energy, but even that will only be using `pw.x` and not `xspectra.x`.

Hope that makes things clearer.

Reply:
Ah, that does clear things up! Thanks a lot for the explanation. All good then!